.github/workflows/pkgprnew.yaml

```diff
@@ -0,0 +1,31 @@
+name: pkg.pr.new
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+
+      - name: Enable Corepack
+        run: corepack enable
+
+      - name: Setup Yarn Berry
+        run: |
+          corepack prepare yarn@3.8.3 --activate
+          yarn set version 3.8.3
```

Comment:
Is there a reason for pinning an exact yarn version here, rather than letting corepack handle it in the usual way (with simply `corepack enable`)?